### PR TITLE
prover9 2009-11A (new formula)

### PR DESCRIPTION
**Pull request by bwesterb**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/prover9.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  url "https://www.cs.unm.edu/~mccune/prover9/download/LADR-2009-11A.tar.gz"
+
+  version "2009-11A"
+  sha256 "c32bed5807000c0b7161c276e50d9ca0af0cb248df2c1affb2f6fc02471b51d0"
+
+  def install
+    ENV.deparallelize
+    system "make", "all"
+    bin.install "bin/prover9", "bin/mace4"
+    man.install Dir["manpages/*"]
```
Comment (fxcoudert) [inline Formula/prover9.rb]:
```suggestion
    man1.install Dir["manpages/*.1"]
```

File: Formula/prover9.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+class Prover9 < Formula
+  desc "Automated theorem prover for first-order and equational logic"
+  homepage "https://www.cs.unm.edu/~mccune/prover9/"
+  url "https://www.cs.unm.edu/~mccune/prover9/download/LADR-2009-11A.tar.gz"
+
+  version "2009-11A"
+  sha256 "c32bed5807000c0b7161c276e50d9ca0af0cb248df2c1affb2f6fc02471b51d0"
+
+  def install
+    ENV.deparallelize
```
Comment (fxcoudert) [inline Formula/prover9.rb]: Has this makefile parallelisation issue been reported?

File: Formula/prover9.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
+class Prover9 < Formula
+  desc "Automated theorem prover for first-order and equational logic"
+  homepage "https://www.cs.unm.edu/~mccune/prover9/"
+  url "https://www.cs.unm.edu/~mccune/prover9/download/LADR-2009-11A.tar.gz"
+
```
Comment (fxcoudert) [inline Formula/prover9.rb]: Remove blank line

File: Formula/prover9.rb [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      formulas(theory).
+      all x all y all z ((x * y) * z = x * (y * z)).
+      exists e ((all x (e * x = x)) &
+                (all x exists y (y * x = e))).
+      exists a exists b (a * b != b * a).
+      end_of_list.
+    EOS
+
+    system bin/"prover9", "-f", testpath/"x2.in"
+    system bin/"mace4", "-f", testpath/"group2.in"
```
Comment (fxcoudert) [inline Formula/prover9.rb]: Good tests!

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/prover9.rb (hunk 4 above)
Comment (bwesterb) [reply Formula/prover9.rb]: Thanks.

File: Formula/prover9.rb (hunk 1 above)
Comment (bwesterb) [reply Formula/prover9.rb]: Applied suggestion.

File: Formula/prover9.rb (hunk 2 above)
Comment (bwesterb) [reply Formula/prover9.rb]: I don't know.  I did not. There is no public issue tracker.

File: Formula/prover9.rb (hunk 3 above)
Comment (bwesterb) [reply Formula/prover9.rb]: Done.

File: Formula/prover9.rb (hunk 2 above)
Comment (SMillerDev) [reply Formula/prover9.rb]: Could you report it please?

Comment (bwesterb) [reply Formula/prover9.rb]: I just did and included a [patch](https://gist.github.com/bwesterb/32361eb82e15db8691d6bf880caf4da8).

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**